test/integration/auth/accessreview_test.go

```diff
@@ -216,7 +198,7 @@ func TestSelfSubjectAccessReview(t *testing.T) {
 			},
 			expectedStatus: authorizationapi.SubjectAccessReviewStatus{
 				Allowed:         false,
-				Reason:          "no",
+				Reason:          "Not in privileged list.\nno",
```

Comment:
I promise to remove this as a reason for the loopback authorizer unless @dims  beats me there.